### PR TITLE
Fix banners

### LINKED ISSUE #2219: (SP: 1) Fix tutor and student banners
### Main Acceptance Criteria:
- [ ] Visual discrepancy must be corrected for the `tutor` banner to ensure consistency with the design specifications outlined in the [Figma design](https://www.figma.com/design/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?node-id=5424-63779&t=Jl4CWkOxBEB0RWH4-4)
- [ ] Visual discrepancy must be corrected for the `student` banner to ensure consistency with the design specifications

The Tutor and Student banners do not align with the [Figma design](https://www.figma.com/design/liJZDYhUnDP15Pi9bipDcv/Space2Study-(Students)?node-id=5424-63779&t=Jl4CWkOxBEB0RWH4-4).

**Steps to reproduce:**
1. Login as `tutor`
2. Navigate to the "`Find offers`" tab.
3. Click on "`All offers`".
4. Toggle "`Students' requests`" or "`Tutors' requests`".
5. Click on "`View details`".
6. Scroll down and observe the tutor/student banner at the top of the screen.

##
**Actual result**:
Screenshot 2024-07-26 at 15 46 50

**Expected result:** 
Screenshot 2024-07-26 at 15 57 04

##
**Actual result**:
Screenshot 2024-07-26 at 15 48 09
**Expected result:** 
Screenshot 2024-07-26 at 16 00 19

### PR DESCRIPTION
**Pull request by Renatavl**
Fixed:
- margins and paddings;
- buttons' sizes;
- "Save offer" button

image

image

### CODE REVIEW

**Review by docia — APPROVED** [review]

File: src/components/offer-banner/OfferBanner.styles.ts [hunk 1]
```diff
@@ -33,20 +33,29 @@ export const styles = {
...
     display: 'flex',
-    gap: '10px'
+    gap: '16px'
   },
   button: {
-    whiteSpace: 'nowrap'
+    whiteSpace: 'nowrap',
+    padding: '14.5px 30px',
+    width: 'auto',
+    lineHeight: '18.96px'
```
[the remark this answers is not in the document]
Comment (Renatavl) [reply src/components/offer-banner/OfferBanner.styles.ts:42]: It was done according to design

[the remark this answers is not in the document]
Comment (abalanovsky) [reply src/components/offer-banner/OfferBanner.styles.ts:42]: It's not okay to use decimals, maybe old mockup

File: src/components/offer-banner/OfferBanner.styles.ts [hunk 2]
```diff
@@ -33,20 +33,29 @@ export const styles = {
...
   },
   bookmarkButton: {
-    color: 'primary.500'
+    color: 'primary.500',
+    padding: '12px 30px',
```
Comment (abalanovsky) [inline src/components/offer-banner/OfferBanner.styles.ts:46]: We are trying to shorten css properties `p`, `m`, `ml` etc

File: src/components/offer-banner/OfferBanner.tsx [hunk 3]
```diff
@@ -33,6 +33,8 @@ const OfferBanner: FC<OfferBannerProps> = ({ offer, buttonActions }) => {
           key={elem.label}
           sx={styles.button}
           {...elem.buttonProps}
+          size={null}
+          style={{ lineHeight: '18.96px' }}
```
Comment (abalanovsky) [inline src/components/offer-banner/OfferBanner.tsx:37]: style to styles

**Review by BohdanMylyi — APPROVED** [review]

**Review by YaroslavChuiko — APPROVED** [review]

**Pull request merged**